compiler/il/OMRNode.hpp

```diff
@@ -31,6 +31,7 @@ namespace OMR { class Node; }
 namespace OMR { typedef OMR::Node NodeConnector; }
 #endif
 
+#include <algorithm>
```

Comment:
I think this is redundant now - please remove if this is the case.

Reply:
Fixed in https://github.com/eclipse/omr/commit/143f78bb3e8b385ae378f14247f8208000fb3665